test/commands/api/update.test.js

```diff
@@ -200,48 +215,65 @@ describe('valid api:update', () => {
       expect(ctx.stdout).to.contains('Updated API org/api/2.0.0')
     })
 
-    test
-      .stub(config, 'getConfig', () => ({ SWAGGERHUB_URL: shubUrl }))
-
-      .nock(`${shubUrl}/apis`, api => api
-        .get('/org/api/settings/default')
-        .reply(200, { version: '2.0.0' })
-      )
-      .nock(`${shubUrl}/apis`, api => api
-        .put('/org/api/2.0.0/settings/private', { private: false })
-        .reply(200)
-      )
-      .stdout()
-      .command(['api:update', 'org/api', '--visibility=public'])
-
-      .it('runs api:update to set API public without a version arg', ctx => {
-        expect(ctx.stdout).to.contains('Updated visibility of API org/api/2.0.0 to public')
-      })
-
-    test
-      .stub(config, 'getConfig', () => ({ SWAGGERHUB_URL: shubUrl }))
-
-      .nock(`${shubUrl}/apis`, api => api
-        .get('/org/api/settings/default')
-        .reply(200, { version: '2.0.0' })
-      )
-      .nock(`${shubUrl}/apis`, api => api
-        .put('/org/api/2.0.0/settings/private', { private: false })
-        .reply(200)
-      )
-      .stdout()
-      .command(['api:update', 'org/api/2.0.0', '--visibility=public'])
-
-      .it('runs api:update to set API public', ctx => {
-        expect(ctx.stdout).to.contains('Updated visibility of API org/api/2.0.0 to public')
-      })
-
-    test
+  test
+    .stub(config, 'getConfig', () => ({ SWAGGERHUB_URL: shubUrl }))
+    .nock(`${shubUrl}/apis`, api => api
+      .get('/org/api/2.0.0/settings/version')
```

Comment:
We see that by passing `visibility=private` we can override the current false value to make it true.